### ISSUE TITLE
Visual glitching with `simple` example

### ISSUE THREAD

**Issue by MrGVSV** [TheRawMeatball/ui4#2]
I just downloaded this project and ran the [`simple`](https://github.com/TheRawMeatball/ui4/blob/446f416a7d6a96f8568409138c6dc8b25572468d/examples/simple.rs) example, but got a lot of visual glitching. Is this a known issue?

**OS:** macOS Big Sur (v11.6)
**rustc:** 1.56.1

https://user-images.githubusercontent.com/49806985/145545927-1fdaef96-3738-4465-ba4b-f9235d80e46c.mp4

**Comment by TheRawMeatball** [comment]
No, this is quite strange. My guess is somehow the buffers bevy_egui uses for rendering have some alignment issues.

Either way, not actionable for me right now, but it should improve once I switch to the new bevy_ui based renderer.
